google-oauth-client-appengine/src/main/java/com/google/api/client/extensions/appengine/auth/oauth2/jakarta/AbstractAppEngineAuthorizationCodeCallbackServlet.java

```diff
@@ -0,0 +1,94 @@
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.api.client.extensions.appengine.auth.oauth2.jakarta;
+
+import com.google.api.client.extensions.servlet.auth.oauth2.jakarta.AbstractAuthorizationCodeCallbackServlet;
+import com.google.appengine.api.users.UserServiceFactory;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import java.io.IOException;
+
+/**
+ * Simple extension of {@link AbstractAuthorizationCodeCallbackServlet} that uses the currently
+ * logged-in Google Account user, as directed in <a
+ * href="https://cloud.google.com/appengine/docs/standard/java/config/webxml#security-auth">Security
+ * and Authentication</a>. This uses the {@code jakarta.servlet} namespace.
+ *
+ * <p>Note that if there is no currently logged-in user, {@link #getUserId(HttpServletRequest)} will
+ * throw a {@link NullPointerException}. Example to require login for all pages:
+ *
+ * <pre>
+ * &lt;security-constraint&gt;
+ * &lt;web-resource-collection&gt;
+ * &lt;web-resource-name&gt;any&lt;/web-resource-name&gt;
+ * &lt;url-pattern&gt;/*&lt;/url-pattern&gt;
+ * &lt;/web-resource-collection&gt;
+ * &lt;auth-constraint&gt;
+ * &lt;role-name&gt;*&lt;/role-name&gt;
+ * &lt;/auth-constraint&gt;
+ * &lt;/security-constraint&gt;
+ * </pre>
+ *
+ * <p>Sample usage:
+ *
+ * <pre>
+ * public class ServletCallbackSample extends AbstractAppEngineAuthorizationCodeCallbackServlet {
+ *
+ * &#64;Override
+ * protected void onSuccess(HttpServletRequest req, HttpServletResponse resp, Credential credential)
+ * throws ServletException, IOException {
+ * resp.sendRedirect("/");
+ * }
+ *
+ * &#64;Override
+ * protected void onError(
+ * HttpServletRequest req, HttpServletResponse resp, AuthorizationCodeResponseUrl errorResponse)
+ * throws ServletException, IOException {
+ * // handle error
+ * }
+ *
+ * &#64;Override
+ * protected String getRedirectUri(HttpServletRequest req) throws ServletException, IOException {
+ * GenericUrl url = new GenericUrl(req.getRequestURL().toString());
+ * url.setRawPath("/oauth2callback");
+ * return url.build();
+ * }
+ *
+ * &#64;Override
+ * protected AuthorizationCodeFlow initializeFlow() throws IOException {
+ * return new AuthorizationCodeFlow.Builder(BearerToken.authorizationHeaderAccessMethod(),
+ * new UrlFetchTransport(),
+ * new GsonFactory(),
+ * new GenericUrl("https://server.example.com/token"),
+ * new BasicAuthentication("s6BhdRkqt3", "7Fjfp0ZBr1KtDRbnfVdmIw"),
+ * "s6BhdRkqt3",
+ * "https://server.example.com/authorize").setCredentialStore(new AppEngineCredentialStore())
+ * .build();
+ * }
+ * </pre>
+ *
+ * @since 1.36.0
```

Comment:
1.36.0 is it the real new version that will be pushed? Otherwise, change it (and other files)

Reply:
Also curious if you need to configured Kokoro cfg files to enable SBOM generation (here and in all Cloud APIs area?)

Reply:
The latest is 1.35.0. This 1.36.0 is the real value to be released.

Reply:
Checking SBOM generation.

Reply:
Confirmed Kokoro configs are set up ([log](http://sponge2/8c8a1384-8db2-4856-b8d3-74e92978d974)). Search for `sbom.spdx.json`.